src/vstart.sh

```diff
@@ -169,6 +170,10 @@ usage=$usage"\t-m ip:port\t\tspecify monitor address\n"
 usage=$usage"\t-k keep old configuration files\n"
 usage=$usage"\t-x enable cephx (on by default)\n"
 usage=$usage"\t-X disable cephx\n"
+usage=$usage"\t-g --gssapi enable Kerberos/GSSApi authentication\n"
+usage=$usage"\t-G disable Kerberos/GSSApi authentication\n"
+usage=$usage"\t-p --ldap enable LDAP authentication\n"
```

Comment:
why do we still have LDAP stuff here? AFAICT, there's no ldap in this PR except for the name of the directory (which should be changed, btw). This option is not even parsed. Please drop this.